src/app/pages/privacy-policy/privacy-policy.component.html

```diff
@@ -0,0 +1,45 @@
+<app-hill-background cover></app-hill-background>
+<div class="home-container">
+  <mat-card class="column">
+    <mat-card-title>
+      <h1 style="font-size:60px;" class="title">Privacy Policy</h1>
+    </mat-card-title>
+    <mat-card-content>
+        <h1>Collection of Information</h1>
```

Comment:
If the title of the page is an `h1` then all subheaders after that should be `h2` and so forth.